addons/social_share/security/ir.model.access.csv

```diff
@@ -0,0 +1,32 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_snshare_campaign_tag,snshare.campaign.tag,model_snshare_campaign_tag,,0,0,0,0
+access_snshare_campaign_tag_user,snshare.campaign.tag.user,model_snshare_campaign_tag,social_share.group_snshare_user,1,1,0,0
+access_snshare_campaign_tag_manager,snshare.campaign.tag.manager,model_snshare_campaign_tag,social_share.group_snshare_manager,1,1,1,1
+
+access_snshare_campaign,snshare.campaign,model_snshare_campaign,,0,0,0,0
+access_snshare_campaign_user,snshare.campaign.user,model_snshare_campaign,social_share.group_snshare_user,1,0,0,0
```

Comment:
I tested with demo and it felt a bit strange ^^ You cannot actually create or write on existing campaigns, so you cannot do much, what do you think?